src/lib.rs

```diff
@@ -392,11 +414,6 @@ impl<T: Counter> Histogram<T> {
         self.total_count
     }
 
-    /// Get the index of the last histogram bin.
```

Comment:
I couldn't think of a public use case for knowing stuff about internal indexes since the value <-> index correspondence is fairly inscrutable, so I made this and a couple of other things non-public.

This function moved down a bit to be with its non-public brethren.